Documentation/Tools/HolographicRemoting.md

```diff
@@ -40,7 +40,7 @@ These issues are particularly relevant when running on **Unity 2019.3** or later
 > [!Note]
 > There is an issue that prevents MSBuild for Unity from functioning properly on some versions of Unity 2019. If **Mixed Reality Toolkit** > **MSBuild** > **Use MSBuild for Unity dependency resolution** does not work correctly, please refer to the [manual installation instructions](#manual-dotnetadapter-installation).
 
-The best way to check is to open Window -> Package Manager and make sure MSBuild for Unity shows up in the packages list. If it's there, assume this step succeeded. If it's not there, try running Mixed Reality Toolkit -> Utilities -> Configure Unity and repeat the steps above for running the MRTK Configurator.
+The best way to check is to open **Window** > **Package Manager** and make sure MSBuild for Unity shows up in the packages list. If it's there, assume this step succeeded. If it's not there, try running **Mixed Reality Toolkit** > **Utilities** > **Configure Unity** and repeat the steps above for running the MRTK Configurator.
```

Comment:
I think that we'll also want to actually add steps to the HoloLens 2 section (i.e. this is now a required step right?)

Reply:
Correct. Will do

Reply:
thanks for pointing this line out.... it is out of date!